tool/travis.sh

```diff
@@ -7,36 +7,44 @@ set -e
 if [ -z "$PKG" ]; then
   echo -e '\033[31mPKG environment variable must be set!\033[0m'
   exit 1
-elif [ -z "$TASK" ]; then
-  echo -e '\033[31mTASK environment variable must be set!\033[0m'
-  exit 1
+fi
+
+if [ "$#" == "0" ]; then
+	echo -e "\033[31mExpected at least one task!\033[0m"
```

Comment:
whitespace! Not a biggy...

Reply:
oh weird apparently vscode uses tabs by default in shell scripts lol